contributors/devel/sig-testing/testing.md

```diff
@@ -189,6 +160,42 @@ This will do the following:
 
 See `go help test` and `go help testflag` for additional info.
 
+### Run unit tests using go test
+
+If you have `GOPATH` set, you can optionally use `go test` to run unit tests.
+
+To run tests for a specific package:
+
+```sh
+go test ./pkg/kubelet
```

Comment:
Out of context from the comparison with the make version, this looks strange. I would be inside of 'pkg/kubelet' and running the test.

For gopath to be relevant this would be `go test k8s.io/kubernetes/pkg/kubelet`.